### DISCUSSION TITLE
How can I sync multiple code snippets?

### DISCUSSION THREAD

**Discussion by CodyBontecou** [slidevjs/slidev#851] [Q&A] [answered]
I want to highlight lines within two code snippets and sync the clicks.

Is this possible with Slidev?

**Comment by DavidAntliff** [comment]
I'm curious about this too - is there a way to send a specific click to multiple elements?

E.g. I have `layout: two-cols-header`, with a `md magic-move` code block on the left, and another on the right (I'm showing the audience how to migrate from one programming language to another), and I want to use the awesome features of magic-move to highlight lines of code in both code blocks simultaneously.

At the moment, multiple magic-move blocks are activated serially, rather than in parallel.

Is there a way to direct single clicks to multiple objects?

**Comment by kermanx** [comment, marked as answer]
See the example here: https://sli.dev/guide/animations#mixed-case
